package.json

```diff
@@ -7,7 +7,10 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && ./node_modules/.bin/babel -sd lib/ src/",
```

Comment:
I don't think you need `./node_modules/.bin/babel` just `babel` afaik npm will add `./node_modules/.bin/` to PATH, when running scripts... though not important :)

Also `-s` is in `babelrc` again nit...